docs/Users_Guide/config_options.rst

```diff
@@ -1626,6 +1626,26 @@ The default setting is >=0.1&&<=0.85.
 
   seeps_p1_thresh = >=0.1&&<=0.85;
 
+seeps_grid_climo_name
+---------------------
+
+The "seeps_grid_climo_name" option sets the location and the filename of the SEEPS climo file for the gridded data.
+The default setting is the empty string. It should be configured by the user. It can be overridden by the environment variable, MET_SEEPS_GRID_CLIMO_NAME.
+
+.. code-block:: none
+
+  seeps_grid_climo_name = "";
+
+seeps_point_climo_name
+----------------------
+
+The "seeps_point_climo_name" option controls controls the location and the filename of the SEEPS climo file for the point data.
```

Comment:
```suggestion
The "seeps_point_climo_name" option controls the location and the filename of the SEEPS climo file for the point data.
```

Reply:
I'll go ahead and commit this change to remove the repeated "controls" in this sentence:

> The "seeps_point_climo_name" option **controls controls** the location and the filename of the SEEPS climo file for the point data.

After I commit that change, I'll go ahead and approve this PR.